smooth/examples/run_smooth_example.py

```diff
@@ -1,11 +1,11 @@
-from smooth.examples.example_model import mymodel
+from smooth.examples.example_model_infeasable import mymodel
```

Comment:
change back to default model

Reply:
https://github.com/rl-institut/smooth/pull/27/commits/f45d31454d1745df1805b852bbdcf5bf0e42c276